test/anacondalib.py

```diff
@@ -70,3 +76,34 @@ def resetStorage(self):
         # CLEAR_PARTITIONS_DEFAULT = -1
         s.dbus_set_initialization_mode(-1)
         s.dbus_scan_devices()
+
+    def __downloadLogs(self):
+        if not self.ext_logging:
+            return
+
+        self.logs_dir = os.path.join('./test_logs', f'{self.__class__.__name__}.{self._testMethodName}')
+        if not os.path.isdir(self.logs_dir):
+            os.makedirs(self.logs_dir)
+
+        self.machine.download('/tmp/anaconda.log', 'anaconda.log', self.logs_dir)
+        self.machine.download('/tmp/packaging.log', 'packaging.log', self.logs_dir)
+        self.machine.download('/tmp/storage.log', 'storage.log', self.logs_dir)
+        self.machine.download('/tmp/dbus.log', 'dbus.log', self.logs_dir)
+        self.machine.download('/tmp/syslog', 'syslog', self.logs_dir)
+        try:
+            self.machine.download('/tmp/anaconda-tb-*', '.', self.logs_dir)
+        except CalledProcessError:
+            pass
+
+    def reboot(self):
```

Comment:
It's slightly confusing to have two definitions of the reboot method. I think this code should be better incorporated with the reboot method in the test/helpers/progress.

Reply:
I've renamed the method to make it less confusing. It does more than what Progress helper should handle.